data/korath/korath ships.txt

```diff
@@ -270,6 +270,77 @@ ship "Korath Raider Mark XLII"
 	description `Constant warfare against surrounding aliens has been building pressure on Korath shipyards to recover some of their ancient prowess. Now, with a fresh industrial network to support their endeavors, the shipwrights of the Korath have updated the design to deal with modern targets.`
 
 
+ship "Korath Ravager"
+	sprite "ship/ravager"
+	thumbnail "thumbnail/ravager"
+	attributes
+		category "Heavy Warship"
+		"cost" 24760000
+		"shields" 32700
+		"hull" 9850
+		"required crew" 193
+		"bunks" 221
+		"mass" 979
+		"drag" 11.3
+		"heat dissipation" .7
+		"fuel capacity" 900
+		"cargo space" 50
+		"outfit space" 943
+		"weapon capacity" 404
+		"engine capacity" 177
+		weapon
+			"blast radius" 300
+			"shield damage" 4400
+			"hull damage" 2200
+			"hit force" 6600
+	outfits
+		"Korath Inferno" 2
+		"Korath Warder"
+		"Korath Fire-Lance" 2
+		"Firelight Missile Bank" 6
+		"Firelight Missile" 120
+
+		"Systems Core (Large)"
+		"Fuel Processor" 2
+		"Plasma Core"
+		"Triple Plasma Core"
+		"Liquid Sodium Cooler"
+		"Korath Repeater Rifle" 219
+
+		"Afterburner (Planetary Class)"
+		"Steering (Lunar Class)"
+		"Thruster (Planetary Class)"
+		"Jump Drive"
+
+	engine -29 96
+	engine 29 96
+	engine 0 90 0.9
+	gun -28 -133.5 "Korath Fire-Lance"
+	gun 28 -133.5 "Korath Fire-Lance"
+	gun -48.5 -85.5 "Firelight Missile Bank"
+		angle 320
+	gun 48.5 -85.5 "Firelight Missile Bank"
+		angle 40
+	gun -145 14.5 "Firelight Missile Bank"
+		angle 250
+	gun 145 14.5 "Firelight Missile Bank"
+		angle 110
+	gun -98 75 "Firelight Missile Bank"
+		angle 190
+	gun 98 75 "Firelight Missile Bank"
+		angle 170
+	turret -32.5 -53.5 "Korath Inferno"
+	turret 32.5 -53.5 "Korath Inferno"
+	turret 0 -23 "Korath Warder"
+	explode "tiny explosion" 120
+	explode "small explosion" 60
+	explode "medium explosion" 70
+	explode "large explosion" 50
+	explode "huge explosion" 15
+	"final explode" "final explosion large"
+	description "While perhaps, at first, seeming like a super sized 'olofez-Class Carried Chaser, the 'nra'ret-Class Chaser is not only the first truly new ship design the Exiles have fielded since the Exodus but also the first purposed directly towards combat usage instead of needing to also double as population and supply housing. Its size and non-modular nature also means it must be constructed wholesale at the largest Korath facilities, something that, until recently, was generally reserved for the likes of the massive Ik'atilajej and the Korsmanath A'awoj."
```

Comment:
```suggestion
	description "While perhaps, at first, seeming like a super-sized 'olofez-Class Carried Chaser, the 'nra'ret-Class Chaser is not only the first truly new ship design the Exiles have fielded since the Exodus but also the first purposed directly towards combat usage instead of needing to also double as population and supply housing. Its size and non-modular nature also means it must be constructed wholesale at the largest Korath facilities, something that, until recently, was generally reserved for the likes of the massive Ik'atilajej and the Korsmanath A'awoj."
```